tests/functional/test_pep517.py

```diff
@@ -6,12 +6,14 @@
 from tests.lib import make_test_finder, path_to_url
 
 
-def make_project(tmpdir, requires=[], backend=None):
+def make_project(tmpdir, requires=[], backend=None, backend_path=[]):
```

Comment:
I would use `None` here to avoid having to think about possible unexpected behavior with mutable defaults.

Reply:
Sure